src/components/Footer/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Footer';
```

Comment:
Though the code is functional, it's better to use a clear import and export to maintain consistency 🧐. Instead of directly re-exporting from the import, import the component and export it separately.